lib/mockbuilder.py

```diff
@@ -65,6 +71,12 @@ def initialize(self):
         utils.run_command(cmd)
 
     def build(self, package):
+        """
+        Build RPM package and its subpackages
+
+        Args:
+            package (str): package name
```

Comment:
This is not a `str`, it's an `RPM_Package` object.
This applies for all methods of this class.